packages/database/src/migrations/20210420031825-WeeklyCasesTrendGraphForPalauCountry-modifies-data.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+import { insertObject } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const DISEASE_DATA_ELEMENTS = {
+  AFR: {
+    dataElement: 'PSSS_AFR_Total_Cases',
+    name: 'Acute Fever and Rash',
+    color: '#F0965BFF', // orange
+  },
+  DIA: {
+    dataElement: 'PSSS_DIA_Total_Cases',
+    name: 'Diarrhoea',
+    color: '#81DEE4FF', // aqua
+  },
+  ILI: {
+    dataElement: 'PSSS_ILI_Total_Cases',
+    name: 'Influenza Like Illness',
+    color: '#4DA347FF', // green
+  },
+  PF: {
+    dataElement: 'PSSS_PF_Total_Cases',
+    name: 'Prolonged Fever',
+    color: '#1C49A7FF', // blue
+  },
+  DLI: {
+    dataElement: 'PSSS_DLI_Total_Cases',
+    name: 'Dengue Like Illness',
+    color: '#8455F6', // purple
+  },
+  CON: {
+    dataElement: 'PSSS_CON_Total_Cases',
+    name: 'Conjunctivitis',
+    color: '#BE72E0', // pink
+  },
+};
+
+const getDashboardReportId = diseaseCode =>
+  `PSSS_PW_${diseaseCode}_Weekly_Case_Trend_Graph_Country`;
+
+const dashboardGroupCode = 'PW_PSSS_Syndromic_Surveillance_National_Data_Country_Public';
+
+const getDashboardReport = (id, diseaseName, dataElementCode, color) => ({
+  id,
+  dataBuilder: 'analyticsPerPeriod',
+  dataBuilderConfig: {
+    dataElementCode,
+    aggregations: [
+      {
+        type: 'FINAL_EACH_WEEK_FILL_EMPTY_WEEKS',
+        config: {
+          fillEmptyPeriodsWith: null,
+        },
+      },
+    ],
+    entityAggregation: {
+      aggregationOrder: 'BEFORE',
+      dataSourceEntityType: 'facility',
```

Comment:
The `dataSourceEntityType` here should be `country`, and there's no need for an `aggregationEntityType`. The indicators will be responsible for converting facility level data to the country level data so the dashboards don't need to worry about that. This is true for other PSSS country level dashboards as well.